cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -214,6 +235,125 @@ public CohortEngineRestHandler() {
 		// NOTE: This constructor is called on every REST call!
 	}
 
+	@POST
+	@Path("/cohort-evaluation")
+	@Consumes(MediaType.MULTIPART_FORM_DATA)
+	@Produces({ MediaType.APPLICATION_JSON })
+	@ApiOperation(value = "Evaluates a specific define within a CQL for a set of patients"
+			, notes = EVALUATION_API_NOTES, response = String.class
+			, tags = {"Cohort Evaluation"}
+			, nickname = "evaluate_cohort"
+			, extensions = {
+			@Extension(properties = {
+					@ExtensionProperty(
+							name = DarkFeatureSwaggerFilter.DARK_FEATURE_NAME
+							, value = CohortEngineRestConstants.DARK_LAUNCHED_COHORT_EVALUATION)
+			})
+	}
+	)
+	@ApiImplicitParams({
+			// This is necessary for the dark launch feature
+			@ApiImplicitParam(access = DarkFeatureSwaggerFilter.DARK_FEATURE_CONTROLLED, paramType = "header", dataType = "string"),
+			// These are necessary to create a proper view of the request body that is all wrapped up in the Liberty IMultipartBody parameter
+			@ApiImplicitParam(name=REQUEST_DATA_PART, value=EXAMPLE_REQUEST_DATA_JSON, dataTypeClass = MeasureEvaluation.class, required=true, paramType="form", type="file"),
+			@ApiImplicitParam(name=CQL_PART, dataTypeClass = File.class, required=true, paramType="form", type="file" )
```

Comment:
The docs here don't line up with what is used inside the method. The name and dataTypeClass are wrong in the first parameter and the name for the second parameter should be  `CQL_DEFINITION`.